news/2023/2023-11-07-winter-fanart-contest.md

```diff
@@ -0,0 +1,53 @@
+- **Entrants must keep a full file containing all of their entry's layers available.** We'll only ask you for this if we think it's needed to ascertain any claims regarding plagiarism or the use of generative art tools. If you don't/can't provide us this within a timely manner when we ask, we'll have no choice but to pull your listing from the contest, sorry!
+- **Artist attribution (names, social media handles, etc.) must be limited to no more than three lines total and confined to any corner of your entry.** Being proud of your work is good, just do it in a reasonably small spot of your work. One thing unique to osu!'s history is that we really don't like forced advertisement, and attribution that is too large can come across like it.
+- **Entries MUST meet the resolution requirements for a main-menu piece — namely 2732x1536. THIS IS CHECKED BY A SCRIPT AND ENTRIES THAT DO NOT MATCH THIS REQUIREMENT WILL BE DISCARDED AUTOMATICALLY.** This remains the \#1 reason for distressed emails sent after the contest about why an entry didn't make it in. Even ONE pixel off is enough. Export your creations at the correct resolution!
+- **Artists should use the custom templates provided ([PSD template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.psd?2017) / [PNG template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.png?2017)) when composing their piece as a guideline to understand where the osu! cookie will be placed on the main menu and during the voting process.** The circular spot in the middle shows where the main-menu cookie will be when inside the game. Factor this in when designing your piece, as anything inside it is likely not going to be seen. This doesn't mean leaving a big circular hole full of nothing in the middle! Be creative. *Make sure you don't leave it visible in your end result.*
+- **Do not leave the dotted osu! cookie template visible in your submission.** Leave the meme in the grave, where it belongs.
+- **If you're using characters from other intellectual properties, do so sparingly.** All the official osu! mascots and the community mascots are fair game, but please avoid using excessive representations of characters from other worlds or franchises in your work — the contest is about osu! foremost, after all. Remember also that the community mascots are someone else's treasured creation, so please treat them with the same respect that you'd treat your own OCs and work.
+
+The [Winter 2023 Fanart Contest listing](https://osu.ppy.sh/community/contests/189) awaits you when you're ready to submit your entry.
+
+And remember, you can resubmit your entry as many times as you like before submissions close. One pixel off? No worries, fix the thing and reupload your entry. Do this thirty times, three hundred times, three THOUSAND times, it's fine! Though if you're doing it that many times, we might wonder just how fine you are, but we digress.
+
+Good luck!
+
+—Ephemeral
```

Comment:
Not sure how past newsposts handled this, but:
The contest listing has the header attribution after the rules, perhaps it should also be here? Credit where credit is due

Reply:
good idea, added